src/main/java/core/basesyntax/service/impl/OrderReturnServiceImpl.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.service.impl;
+
+import static core.basesyntax.Constants.FRUIT_NAME_ARRAY_INDEX;
+
+import core.basesyntax.handler.OrderReturnHandler;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.DataParserService;
+import core.basesyntax.service.OrderReturnService;
+import java.util.List;
+
+public class OrderReturnServiceImpl implements OrderReturnService {
```

Comment:
Or better to create 
```suggestion
public class ReturnOperationHandler implements OperationHandler {
```

Reply:
What about ReturnOperationService? Becouse i have handler already